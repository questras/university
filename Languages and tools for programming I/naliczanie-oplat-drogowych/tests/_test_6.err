Error in line 7: Car200 A200 7,0
Error in line 8: Car200 S200 5,0
Error in line 9: Car200 A200 4,0
Error in line 15: Car302 S301 1,0
Error in line 14: Car301 A301 1,0
Error in line 24: C1 A1 0,0
Error in line 25: C1 A2 0,0
Error in line 26: C1 A1 1,0
Error in line 28: C500 A501 0,0
Error in line 29: C500 A502 0,0
