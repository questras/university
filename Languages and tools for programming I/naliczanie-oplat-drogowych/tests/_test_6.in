Car100 A100 7,0
Car100 A100 6,0
Car100 A100 14,0
Car100 A100 11,0
?A100

Car200 A200 7,0
Car200 S200 5,0
Car200 A200 4,0
Car200 S200 7,0
?A200
?S200

Car301 A301 1,0
Car302 S301 1,0
Car302 S302 1,0
Car301 A302 1,0
Car303 S307 1,0

Caryca A400 3,0
Caryca A400 3,0
?Caryca

C1 A1 0,0
C1 A2 0,0
C1 A1 1,0

C500 A501 0,0
C500 A502 0,0
C500 A501 1,0
