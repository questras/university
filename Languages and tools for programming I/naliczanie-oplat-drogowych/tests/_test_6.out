A100 4,0
Caryca A 0,0
